shopfloor/tests/test_actions_data_detail.py

```diff
@@ -202,6 +204,18 @@ def test_data_move_line_package(self):
             "progress": 30.0,
         }
         self.assertDictEqual(data, expected)
+        data = self.data_detail.move_line(move_line, with_package_move_line_count=True)
+        package_src = expected["package_src"]
+        package_src.update({"move_line_count": 1})
+        package_dest = expected["package_dest"]
+        package_dest.update({"move_line_count": 1})
+        expected.update(
+            {
+                "package_src": package_src,
+                "package_dest": package_dest,
+            }
+        )
```

Comment:
It makes sense because is already tested above that the pkgs are correct. Yet, this seems too verbose.
What about this?
```suggestion
        expected["package_src"]["move_line_count"] = 1
        expected["package_dest"]["move_line_count"] = 1
```
That's all you need IMO. Same in the other tests.

Reply:
yes was already change not yet commited :-)